src/tqec/plaquette/library/css.py

```diff
@@ -8,64 +8,97 @@
 
 from tqec.circuit.qubit_map import QubitMap
 from tqec.circuit.schedule import ScheduledCircuit
-from tqec.plaquette.enums import MeasurementBasis, ResetBasis
+from tqec.circuit.moment import Moment, iter_stim_circuit_without_repeat_by_moments
+from tqec.plaquette.enums import MeasurementBasis, PlaquetteSide, ResetBasis
 from tqec.plaquette.plaquette import Plaquette
 from tqec.plaquette.qubit import SquarePlaquetteQubits
 
 
 def make_css_surface_code_plaquette(
     basis: Literal["X", "Z"],
-    data_qubits_initialization: ResetBasis | None = None,
-    data_qubits_measurement: MeasurementBasis | None = None,
+    data_initialization: ResetBasis | None = None,
+    data_measurement: MeasurementBasis | None = None,
     x_boundary_orientation: Literal["HORIZONTAL", "VERTICAL"] = "VERTICAL",
+    init_meas_only_on_side: PlaquetteSide | None = None,
 ) -> Plaquette:
-    """Create a CSS-type surface code plaquette.
+    """Make a CSS-type surface code plaquette.
 
     Args:
-        basis: The basis of the plaquette, either "X" or "Z".
-        data_qubits_initialization: Initialization basis for the data qubits.
-            If None, no initialization is performed.
-        data_qubits_measurement: Measurement basis for the data qubits.
-            If None, no measurement is performed.
-        x_boundary_orientation: The orientation of the X boundary of the surface
-            code block. Either "HORIZONTAL" or "VERTICAL". This determines the
-            CNOT order in the plaquette together with the basis to prevent hook
-            error from decreasing the code distance. Default is "HORIZONTAL".
+        basis: the basis of the plaquette.
+        x_boundary_orientation: the orientation of the X boundary.
+        data_initialization: the logical basis for data initialization.
+        data_measurement: the logical basis for data measurement.
+        init_meas_only_on_side: the side for data initialization and measurement.
```

Comment:
```suggestion
        basis: the basis of the plaquette.
        data_initialization: the logical basis for data initialization.
        data_measurement: the logical basis for data measurement.
        x_boundary_orientation: the orientation of the X boundary.
        init_meas_only_on_side: the side for data initialization and measurement.
```
ordering of arguments in docstring